kolibri/core/content/utils/channel_import.py

```diff
@@ -240,6 +240,7 @@ def __init__(
     ):
         self.channel_id = channel_id
         self.channel_version = channel_version
+        self.current_channel = ChannelMetadata.objects.get(id=self.channel_id)
```

Comment:
You'll want to wrap this in the `try/except` like its source, because if the channel hasn't been imported yet, this will throw `ChannelMetadata.DoesNotExist` error.